packages/deployment/scripts/deploy-gov-v2.ts

```
@@ -0,0 +1,221 @@
+    }
+
+    try {
+      console.log(kleur.cyan().bold(`\nExecuting ${step}...\n`))
+
+      switch (step) {
+        case STEPS.SYSTEM:
+          await systemGov(config, useBummerConfig)
+          break
+        case STEPS.VERIFY:
+          console.log(kleur.yellow().bold('\nVerifying contracts...\n'))
+          const chainId = hre.network.config.chainId
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)